nss-day-cohort-56/worst-website-ever-team-2-best-worst-website
# added styles directory with main.css

# How to test

1.  Pull branch gam-make-css-folder from github
2.  Use ls command to see if styles containing main.css exists
3.  Merge with main
